docs/openapi/components/schemas/credentials/PetroleumAndNaturalGasTitleCredential.yml

```diff
@@ -0,0 +1,141 @@
+$linkedData:
+  term: PetroleumAndNaturalGasTitleCredential
+  '@id': https://w3id.org/traceability#PetroleumAndNaturalGasTitleCredential
+title: Petroleum And Natural Gas Title
+tags:
+  - Other
+description: |
+  The majority of subsurface petroleum and natural gas (PNG) resources in British Columbia (B.C.) are owned by the Province. By entering into a tenure agreement with the Province, private industry can develop these resources. Tenure agreements are the mechanism used by the Province to give rights to petroleum and natural gas resources through issuance of Petroleum and Natural Gas Titles.
+type: object
+required:
+  - '@context'
+  - id
+  - type
+  - issuanceDate
+  - issuer
+  - credentialSubject
+properties:
+  '@context':
+    type: array
+    readOnly: true
+    default:
+      - https://www.w3.org/2018/credentials/v1
+      - https://w3id.org/traceability/v1
+    items:
+      - type: string
+        const: https://www.w3.org/2018/credentials/v1
+    additionalItems:
+      type: string
+      enum:
+        - https://w3id.org/traceability/v1
+  type:
+    type: array
+    readOnly: true
+    default:
+      - VerifiableCredential
+      - PetroleumAndNaturalGasTitleCredential
+    const:
+      - VerifiableCredential
+      - PetroleumAndNaturalGasTitleCredential
+    items:
+      type: string
+      enum:
+        - VerifiableCredential
+        - PetroleumAndNaturalGasTitleCredential
+  id:
+    type: string
+    format: uri
+  issuanceDate:
+    type: string
+    format: date-time
+  issuer:
+    type: object
+    properties:
+      id:
+        type: string
+        format: uri
+  credentialSubject:
+    type: object
+    properties:
+      type:
+        type: array
+        items:
+          type: string
+          enum:
+            - PetroleumAndNaturalGasTitle
+      title_number:
```

Comment:
snake_casing really burns my eyes... We use camelCase consistently everywhere else.